semantic_release/helpers.py

```diff
@@ -114,7 +114,7 @@ def parse_git_url(url: str) -> ParsedGitUrl:
     # Normalizers are a list of tuples of (pattern, replacement)
     normalizers = [
         # normalize implicit ssh urls to explicit ssh://
-        (r"^(\w+@)", r"ssh://\1"),
+        (r"^([\w._-]+@)", r"ssh://\1"),
```

Comment:
This change to add other special characters to the user name of an implicit ssh URL should also extend to the scp syntax converter below. As after this lines pass, it will now have a `ssh://` in front of it. The point of the next line is to convert the `:` to a `/` so that the ssh url looks like a normal HTTP url for parsing components.

If your unit test worked without this in a bit surprised.

Reply:
I made the change which allows the tests to pass.